docs/api/instruments/tti/index.rst

```diff
@@ -0,0 +1,13 @@
+.. module:: pymeasure.instruments.tti
+
+###########################
+Thurlby Thandar Instruments
+###########################
+
+
+This section contains specific documentation on the Thurlby Thandar Instruments (TTi), Thurlby and Thandar instruments that are implemented. If you are interested in an instrument not included, please consider :doc:`adding the instrument </dev/adding_instruments/index>`.
```

Comment:
That does not sound good: `Thurlby Thandar Instruments (TTi), Thurlby and Thandar instruments`. I do not understand, what you want to say.

Reply:
Thurlby & Thandar were two separate companies. At some point they merged to form a new company "Thurlby Thandar Instruments" (TTi).

How about I rename the directory `thurlby` and if, in the future, someone wants to add a Thandar or TTi instrument then they can create a directory called `thandar` or `tti`?

Reply:
"tti" is fine.
As I did not know that history, I got confused by that phrasing.
You could use something like, `Thurlby Thandar Instruments (TTi) or one of their predecessors Thurlby Instruments and Thandard Instruments`.